docs/buildah-build.md

```diff
@@ -38,6 +38,10 @@ Add a custom host-to-IP mapping (host:ip)
 
 Add a line to /etc/hosts. The format is hostname:ip. The **--add-host** option can be set multiple times.
 
+**--all-platforms**
+
+Instead of building for a set of platforms specified using the **--platform** option, inspect the build's base images, and build for all of the platforms for which they are all available.  Stages which use *scratch* as a starting point can not be inspected, so at least one non-*scratch* stage must be present for detection to work usefully.
```

Comment:
Minor nit/suggestion
```suggestion
Instead of building for a set of platforms specified using the **--platform** option, inspect the build's base images, and build for all of the platforms for which they are all available.  Stages that use *scratch* as a starting point can not be inspected, so at least one non-*scratch* stage must be present for detection to work usefully.
```

Reply:
Changed.